### DISCUSSION TITLE
Rock 4 i2c baud?

### DISCUSSION THREAD

**Discussion by AMuellerAtAHS** [MichaIng/DietPi#6254] [Q&A]
Hi,

can anybody tell me [1] which baud i2c-(7) on radxa rock 4 uses? And [2] how I am able to change that?

**Comment by MichaIng** [comment]
Double posting:

You'd need to adjust the device tree, or better create an own device tree overlay to enable i2c7 but also change its frequency. Could you check:
```sh
ls -l /proc/device-tree/i2c@ff160000/
cat /proc/device-tree/i2c@ff160000/*frequency*
```
